### PR TITLE
Rich Text Input  

### PR DESCRIPTION
**Pull request by NileshArnaiya**
Markdown with Draft WYSIWYG editor

### CODE REVIEW

File: webapp/src/components/form/FormRichText.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+class EditorContainer extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      editorState: EditorState.createEmpty(),
+    };
+  }
+  onEditorStateChange = (editorState) => {
+    // console.log(editorState)
```
Comment (avishkar58) [inline webapp/src/components/form/FormRichText.js]: Remove

**Comment by avishkar58** [comment]
Looks good to me, but let's wait for @KaleabTessera to review.

**Comment by avishkar58** [comment]
Suggestion: try add this to applicationForm to test that it works as a component in the form.

**Comment by NileshArnaiya** [comment]
So, I tried replacing FormTextArea with FormRichText and it looks good on the UI part but I'm not sure how to move ahead with the validation part. How do I add the same validation that is for FormTextArea to FormRichText? Thanks for guiding me.

**Comment by KaleabTessera** [comment]
@NileshArnaiya Can you try having FormRichText exntend FormTextArea (inheritance) and then overwrite some of functionality? That way you can just use the validation alreadty built into FormTextArea.

**Comment by NileshArnaiya** [comment]
Copy pasted functions from FormTextArea turns out those same functions wouldn't work for the EditorState FormRicH in DraftJS, same for showError and errorText in ApplicationForm, also wordcount needs to be fetched differently for it.

**Pull request closed without merging**